dingo/pipe/sampling.py

```diff
@@ -29,7 +29,7 @@ def __init__(self, args, unknown_args):
         self.ini = args.ini
         self.scheduler = args.scheduler
         # self.periodic_restart_time = args.periodic_restart_time
-        self.request_cpus = args.request_cpus
+        self.request_cpus = args.request_cpus_sampling
```

Comment:
I think you need to reverse this line as well.
```suggestion
        self.request_cpus = args.request_cpus
```